dcos/util.py

```diff
@@ -654,6 +654,19 @@ def validate_png(filename):
         raise DCOSException(
             'Unable to validate [{}] as a PNG file'.format(filename))
 
+def get_nondefault_service_name():
+    """
+
+    :rtype: str or None
+
+    """
+
+    if constants.DCOS_SERVICE_NAME_ENV in os.environ:
+        return os.environ[constants.DCOS_SERVICE_NAME_ENV]
+    else:
+        return None
+
```

Comment:
if we add an `env var` we need to also add a config variable for consistency. see https://github.com/dcos/dcos-cli/blob/master/dcos/config.py#L59